static/compiler.js

```diff
@@ -73,6 +73,7 @@ function Compiler(hub, container, state) {
     this.compilerService = hub.compilerService;
     this.domRoot = container.getElement();
     this.domRoot.html($('#compiler').html());
+    this.hideable = this.domRoot.find('.hideable');
```

Comment:
This might be better off on `Editor.initButtons`

Reply:
Good spot! Thanks!